js-callback-functions_user-login-callback-4-with-multiple-functions/js/index.js

```diff
@@ -9,3 +9,14 @@ function handleUserLogin(onSuccess, onError, userName) {
 }
 
 // The exercise starts here!
+function showWelcomeMessage(userName, userRole) {
+  console.log(`Welcome ${userName}! You are logged in now as ${userRole}.`);
+}
+
+function showErrorMessage(errorMessage) {
+  console.log(`Login error: ${errorMessage}`);
+}
+
+handleUserLogin(showWelcomeMessage, showErrorMessage, (userName = "Jane Doe"));
```

Comment:
you can also just write the 'username' as a string without parens = :)

Reply:
I guess this was prettier 🤔